src/shilpa/Assignment_15/BankAccountsUsingSwitchCase.java

```diff
@@ -0,0 +1,160 @@
+package shilpa.Assignment_15;
+
+import java.util.Scanner;
+public class BankAccountsUsingSwitchCase {
+	int iciciBal, hdfcBal, bobBal, sbiBal;
+	String bankname;
+	int iciDebitCnt = 0, hdfcDebitCnt = 0, bobDebitCnt = 0, sbiDebitCnt = 0;
+	int iciCreditCnt = 0, hdfcCreditCnt = 0, bobCreditCnt = 0, sbiCreditCnt = 0;
+	int totalDebitAttempt = 0;
+	int totalCreditAttempt = 0;
+
+	public void debitCreditOps(String operation, String bank, int amt) {
+		bankname=bank;
+		switch (operation.toUpperCase()) {
+		
+//******************* Debit Operation **********************
+		case "DEBIT":
+			switch (bankname.toUpperCase()) {
+			case "ICICI":
+				if (iciciBal >= amt) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.